releaf-core/app/assets/stylesheets/releaf/include/error.scss

```diff
@@ -16,18 +14,17 @@ section.error h1 i
     color: #ef3d64;
     font-size: 25px;
     position: absolute;
-    left: -62px;
-    top: 14px;
+    left: 0px;
+    top: 17px;
     margin-bottom: 11px;
 }
 
 section.error .description
 {
     color: #616161;
-    max-width: 500px;
     font-size: 14px;
     line-height: 25px;
-    margin: 0 0 18px 0;
+    margin: 18px 0 18px 0;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Shorthand form for property `margin` should be written more concisely as `18px 0` instead of `18px 0 18px 0`